dist/stat.py

```diff
@@ -143,7 +143,7 @@ def print_func(name, handle, list):
 __wt_stat_''' + name + '_clear_single(WT_' + name.upper() + '''_STATS *stats)
 {
 ''')
-    for l in sorted(list):
+    for l in statlist:
```

Comment:
We used to sort all of the lists again here, into a different order.  I've changed the input to just sort them once and generate everything consistently from that list.